app/src/main/res/layout/item_autocomplete_bookmark_suggestion.xml

```diff
@@ -28,7 +28,7 @@
         android:layout_height="24dp"
         android:layout_marginStart="16dp"
         android:importantForAccessibility="no"
-        android:src="@drawable/ic_bookmark_24dp"
+        android:src="@drawable/ic_autocomplete_bookmark_24dp"
```

Comment:
Does it make sense to keep the 24dp suffix? This usually is created when using the Asset Creator from Android Studio and I think we could just get rid of it

Reply:
Good thinking. Done!